php/class-coauthors-wp-list-table.php

```diff
@@ -249,17 +249,28 @@ function column_linked_account( $item ) {
 	 * Render the published post count column
 	 */
 	function column_posts( $item ) {
+		$count = $this->get_post_count( $item );
+		return '<a href="' . esc_url( add_query_arg( 'author_name', rawurlencode( $item->user_login ), admin_url( 'edit.php' ) ) ) . '">' . $count . '</a>';
+	}
+
+	/**
+	 * Get the post count for the guest author
+	 *
+	 * @global oebject $coauthors_plus
+	 * @param object $item guest-author object.
+	 * @return int post count for the guest author
+	 */
+	function get_post_count( $item ) {
```

Comment:
I would suggest moving this function to the `CoAuthors_Plus` class for wider use, instead of having to instantiate a `CoAuthors_WP_List_Table` class.